tests/installation/confirm_installation.pm

```diff
@@ -9,9 +9,21 @@
 
 use strict;
 use warnings;
+use testapi;
 use base 'y2_installbase';
+use version_utils 'is_sle';
 
 sub run {
+    # For SLE 15 SP4 tests that have the Legacy module added during instalaltion, there is
+    # additional licence popup that is handled by the following function.
+    if ((get_var("SCC_ADDONS") =~ /legacy/) && (is_sle("=15-SP4"))) {
+        my $package_license_popup = $testapi::distri->get_accept_popup_controller();
+        $package_license_popup->wait_accept_popup({
+                timeout => 3000,
```

Comment:
the confirm package popup happens intermediately, there is not need to have an explicit wait.

Reply:
can you share the link? what is the timeout and how much needs in the slower arch? 3000 is too much, and we should not abuse of this mechanism of explicit wait.

Reply:
btw your runs are done in your machine, for timeout I would recommend OSD, we might find something funny...

Reply:
I am running some VRs on OSD now. For the failed runs without the timeout , it might be this one: http://falafel.suse.cz/tests/901#step/confirm_package_license/2 or it has desappeared.

Reply:
that is some other failure during your development, I don't think is needed extra wait, we already wait 2 mins iirc,
`Can't call method "exist" on an undefined value`

Reply:
ok you added to the ticket (just noticed), so I'm fine if we do in the follow-up.